### ISSUE TITLE
Issue with passing an array of paths

### ISSUE THREAD

**Issue by anyong** [alex-whitney/express-promise-router#12]
This works with the default express.Router() and fails with express-promise-router:

```js
homeRouter.get(['/:language', '/'], function(req, res) {
  if (req.params.language != null) {
    req.lang = req.params.language;
  }
  return res.render('index');
});
```

The `/` URL just says `Cannot GET /`, and another URL like `/en` for example gives the following stacktrace:

```
TypeError: undefined is not a function
   at handleReturn (c:\dev\project.js\node_modules\express-promise-router\lib\express-promise-router.js:27:27)
   at c:\dev\project.js\node_modules\express-promise-router\lib\express-promise-router.js:55:9
   at Layer.handle [as handle_request] (c:\dev\project.js\node_modules\express\lib\router\layer.js:82:5)
   at next (c:\dev\project.js\node_modules\express\lib\router\route.js:110:13)
   at Route.dispatch (c:\dev\project.js\node_modules\express\lib\router\route.js:91:3)
   at Layer.handle [as handle_request] (c:\dev\project.js\node_modules\express\lib\router\layer.js:82:5)
   at c:\dev\project.js\node_modules\express\lib\router\index.js:267:22
   at param (c:\dev\project.js\node_modules\express\lib\router\index.js:340:14)
   at param (c:\dev\project.js\node_modules\express\lib\router\index.js:356:14)
   at Function.proto.process_params (c:\dev\project.js\node_modules\express\lib\router\index.js:400:3)
   at next (c:\dev\project.js\node_modules\express\lib\router\index.js:261:10)
   at c:\dev\project.js\build\webpack:\server\routes\home.coffee:16:5
   at handleReturn (c:\dev\project.js\node_modules\express-promise-router\lib\express-promise-router.js:27:27)
   at c:\dev\project.js\node_modules\express-promise-router\lib\express-promise-router.js:55:9
   at Layer.handle [as handle_request] (c:\dev\project.js\node_modules\express\lib\router\layer.js:82:5)
   at trim_prefix (c:\dev\project.js\node_modules\express\lib\router\index.js:302:13)
   at c:\dev\project.js\node_modules\express\lib\router\index.js:270:7
   at Function.proto.process_params (c:\dev\project.js\node_modules\express\lib\router\index.js:321:12)
   at next (c:\dev\project.js\node_modules\express\lib\router\index.js:261:10)
   at c:\dev\project.js\node_modules\i18n-future\lib\middleware.js:43:7
   at process._tickCallback (node.js:355:11)
```

**Comment by TheThing** [comment]
Yeah, looks like express-promise-router only supports string or regexp as path, see here: https://github.com/alex-whitney/express-promise-router/blob/master/lib/express-promise-router.js#L70

Something a simple PR won't be able to fix though :)

**Comment by anyong** [comment]
Wasn't this fixed by that PR? It's not working for me, has it not been pushed yet?

**Comment by TheThing** [comment]
Nope, it hasn't been pushed yet as far as I can tell.

Your best bet would be to directly reference the github in the project.json file.

**Comment by anyong** [comment]
That's what I've done already, thanks.

**Comment by TheThing** [comment]
You might be interested in using this one: https://github.com/TheThing/express-promise-router
It has a cleaned and optimised code.

Here's the PR with details: https://github.com/alex-whitney/express-promise-router/pull/23

**Comment by alex-whitney** [comment]
This should be fixed in 1.1.0
